forte/data/multi_pack.py

```diff
@@ -923,18 +920,15 @@ def deserialize(
         return mp
 
     @classmethod
-    def from_string(cls, data_content: str):
+    def from_string(cls, data_content: str, json_method: str = "json"):
         # pylint: disable=protected-access
         # can not use explict type hint for mp as pylint does not allow type change
         # from base_pack to multi_pack which is problematic so use jsonpickle instead
-
-        mp = jsonpickle.decode(data_content)
-        if not hasattr(mp, "pack_version"):
-            mp.pack_version = DEFAULT_PACK_VERSION
+        mp = super().from_string(data_content, json_method)
```

Comment:
Should this contain a check of whether `json_method` is `jsonpickle` or `json`?

Reply:
[`super().from_string()`](https://github.com/asyml/forte/pull/917/files#diff-08f2c9d027629fa160cbf4bbd3d8df70b58889cdd88632b3e369126c1c84d447R245) will check `json_method` and then handle the deserialization accordingly.